### PR TITLE
Add CCS811 sensor configuration

### PR DESCRIPTION
**Pull request by silkeh**
This adds configuration for the CCS811 sensor based on the schematics of the board.

### CODE REVIEW

File: dmn_meetstation/include/board.h [hunk 1]
```diff
@@ -25,6 +25,23 @@
 extern "C" {
 #endif
 
+/**
+ * @name    Sensor configuration
+ * @{
```
Comment (bergzand) [inline dmn_meetstation/include/board.h:30]: Missing closing tag

File: dmn_meetstation/include/board.h [hunk 2]
```diff
@@ -25,6 +25,23 @@
 extern "C" {
 #endif
 
+/**
+ * @name    Sensor configuration
```
Comment (bergzand) [inline dmn_meetstation/include/board.h]: Maybe call this `CCS811 eCO2 sensor configuration`

**Review by bergzand — CHANGES_REQUESTED** [review]

**Comment by bergzand** [comment]
Feel free to squash immediately

File: dmn_meetstation/include/board.h (hunk 2 above)
Comment (silkeh) [reply dmn_meetstation/include/board.h]: I've used 'gas sensor' as per https://ams.com/ccs811

**Review by bergzand — APPROVED** [review]

**Pull request merged**